lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -336,3 +369,227 @@ def _delete_node(node_ctx: NodeContext, wait_delete: bool) -> None:
                 for node in environment.nodes.list()
             ]
         )
+
+
+class HyperVDevicePool(BaseDevicePool):
+    SUPPORTED_POOL_TYPE = [
```

Comment:
Supported pool type should be platform specific. Platform should decide pool type it supports. As of now we dont have a specific case for it but keeping it at platform level would make more sense as it would be isolated for both the platform

Reply:
You can have the member in BaseClass and override it in int init of platform specific classes.
This way you can do something like `self.supported_pool_type` and avoid parameter to `configure_device_passthrough_pool`

Reply:
Done